bower.json

```diff
@@ -25,7 +25,9 @@
     "angular-hotkeys": "chieffancypants/angular-hotkeys#~1.4.3",
     "favico.js": "~0.3.5",
     "seedrandom": "~2.3.11",
-    "decimal.js": "git://github.com/MikeMcl/decimal.js.git#~4.0.1"
+    "decimal.js": "git://github.com/MikeMcl/decimal.js.git#~4.0.1",
+    "dropbox-datastores-1.0-latest": "https://www.dropbox.com/static/api/dropbox-datastores-1.0-latest.js",
```

Comment:
According to eth doco yes..
 
Usage: Include dropbox javascript sdk, angular.js and dropstore-ng.js in your application.
```
<script src="https://www.dropbox.com/static/api/dropbox-datastores-1.0-latest.js" type="text/javascript"></script>
<script src="//ajax.googleapis.com/ajax/libs/angularjs/1.0.7/angular.min.js"></script>
<script src="dropstore-ng.js"></script>
```
 
 
From: Evan Rosson [mailto:notifications@github.com] 
Sent: 05 February 2015 02:11
To: erosson/swarm
Cc: David Pyke Le Brun
Subject: Re: [swarm] first impl of dropbox saving/loading. (#260)
 
In bower.json <https://github.com/erosson/swarm/pull/260#discussion_r24138743> :
> @@ -25,7 +25,9 @@
>      "angular-hotkeys": "chieffancypants/angular-hotkeys#~1.4.3",
>      "favico.js": "~0.3.5",
>      "seedrandom": "~2.3.11",
> -    "decimal.js": "git://github.com/MikeMcl/decimal.js.git#~4.0.1"
> +    "decimal.js": "git://github.com/MikeMcl/decimal.js.git#~4.0.1",
> +    "dropbox-datastores-1.0-latest": "https://www.dropbox.com/static/api/dropbox-datastores-1.0-latest.js",
Is this still needed with dropstore-ng?
—
Reply to this email directly or view it on GitHub <https://github.com/erosson/swarm/pull/260/files#r24138743> .  <https://github.com/notifications/beacon/AAX4xEXQCFNtOz78WthHFSuqg11Tz1l3ks5nosi1gaJpZM4DaV1G.gif>